docs/New Game.md

```diff
@@ -31,14 +31,82 @@ game to have proper pickups:
+Up to now, everything is just theoretical values inside Randovania. The next step is making these modifications be
+applied to the game. 
 
-In order to export a game, a patcher is required. Typically, the patcher will be included in Randovania for seamless exports. This is a requirement for non-experimental games. Experimental games, however, may prefer an alternative implementation: refer to Corruption as an example of exporting data to be passed to a standalone patcher.
+In most case, this involves readings the files in
+a user-provided copy of the game and creating either a new ROM/ISO file or a mod file.
 
-**TODO**: details
+This process is split into two parts:
+
+### Patch Data Factory
+
+Each game defines a class that inherits from `PatchDataFactory`. Given a LayoutDescription, a CosmeticPatches and 
+PlayersConfiguration, it creates a JSON Object that will be passed as an argument to the Game Exporter.
+
+### Game Exporter
+
+Each game also defines a class that inherits from `GameExporter`. Given the JSON created by the PatchDataFactory and a
+GameExportParams, provided by your game's GameExportDialog, it's expected to create all the necessary files for the
+user to play the game.
+
+Since modifying the game files requires nothing from Randovania and can be a quite big task, a separate python package 
+responsible for this task is created. This is called the patcher for the game. You are free to implement this package
+however you want, but we recommend the following:
+
+#### Input Data
+
+The arguments for the patcher should be:
+- A Json file, called the patcher data.
+- Paths to needed game files
+- Paths to where to output goes.
+- A status update function
+
+Nothing more. Any other parameter should come from the json file.
+
+#### Language
+
+Best:
+- Python
+
+Good (Known to integrate without issues):
+- C
+- C++
+- Rust
+- Lua
+
+Likely Good (Should integrate well, never tested):
+- Languages that compile to static library, such a Go
+
+Ok (Expect additional work to integrate properly, avoid if viable):
+- C#
+
+Avoid languages that require the user to install a runtime, such as Java.
+
+#### Standalone Use
+
+Allow your patcher to be used directly in the command line, by passing the game files and a json file.
+
+This makes it easy to test the patcher without having to do it through Randovania, and helps identify issues better.
+
+#### Json Schema
+
+Define a [JSON Schema](https://json-schema.org/) that defines the structure of the patcher data. This makes for a good
+documentation of what is the input for the patcher, as well as a very good safety against misconfiguration in Randovania. 
 
 
 ### Multiworld
 
 **TODO**
+
+# Rules for new games
+
+Randovania has a high bar of quality for our game integrations, and we wish to keep it that way. In order to help
+with that goal, there are rules that need to be met before we can accept a game.
+
+- An owner for the game, with an expectation to be available for a while.
```

Comment:
An owner for the game. The owner is expected to be available and willing to maintain their game for the foreseeable future.